docs/source/modules/convection_simple_betts_miller.rst

```diff
@@ -1,16 +1,16 @@
-Convection scheme: Simple Betts-Miller
+Convection scheme: Simple Betts--Miller
 =================================
 
 
 Summary
 -------
-The simplified Betts-Miller (SBM) scheme is a moist adjustment convection scheme developed by [Frierson2007]_ and the motifications described in [OGormanSchneider2008]_. The SBM scheme is Isca's default convection scheme. The key purpose of the routine is output a moisture tendency ``deltaq`` and a temperature tendency ``deltaT``. 
+The simplified Betts--Miller (SBM) scheme is a moist adjustment convection scheme developed by [Frierson2007]_ and the modifications described in [OGormanSchneider2008]_. The SBM scheme is Isca's default convection scheme. The key purpose of the routine is to output a moisture tendency ``deltaq``, a temperature tendency ``deltaT``, and a convective raincall amount ``rain``. The scheme has both deep convection and non-precipitating shallow convection.
```

Comment:
* I think it should be `Betts-Miller` (with one dash)
* Typo: "convective rainfall"

Reply:
Thanks.